packages/client/src/modules/token-set.ts

```diff
@@ -26,11 +26,11 @@ export default class TokenSet {
     return this.expiresIn === 0;
   }
 
-  public claims(): IDToken {
+  public claims(): IdTokenClaims {
```

Comment:
will it be better to use [property getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)?

Reply:
I believe there should be a separate PR related to the tokenSet refactor. cc: @IceHe

Reply:
@IceHe is it true? please paste linear issue number here before resolving the conversation

Reply:
- https://linear.app/silverhand/issue/LOG-1272/browser-refactor
    - https://linear.app/silverhand/issue/LOG-1273/refactor-tokenset

Reply:
mind to elaborate LOG-1273 in case we forget the context? you can specify what to refactor or paste PR comment link

Reply:
Done